<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerDetails</class>
 <widget class="QDialog" name="ControllerDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/qutescoop.png</normaloff>:/icons/qutescoop.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Controller Information</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblControllerInfo">
        <property name="text">
         <string>&lt;strong&gt;ATC: C3 Donald Fauntleroy Duck&lt;/strong&gt;&lt;br&gt;On IVANIT for 01:23, IvAc 1.2.3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="lblStationInformatoin">
          <property name="text">
           <string>Approach on frequency &lt;strong&gt;123.45&lt;/strong&gt;&lt;br&gt;Voice: de.ts.ivao.aero/LOVV_CTR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnJoinChannel">
          <property name="text">
           <string>Join Channel</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>ATIS</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblAtis">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>422</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonShowOnMap">
       <property name="text">
        <string>&amp;Show on Map</string>
       </property>
       <property name="shortcut">
        <string>S</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddFriend">
       <property name="text">
        <string>Add Friend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonShowOnMap</sender>
   <signal>clicked()</signal>
   <receiver>ControllerDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ControllerDetails</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
